src/main/drivers/pwm_output_dshot.c

```diff
@@ -85,8 +84,44 @@ void pwmWriteDigital(uint8_t index, uint16_t value)
     for (int i = 0; i < 16; i++) {
         motor->dmaBuffer[i] = (packet & 0x8000) ? MOTOR_BIT_1 : MOTOR_BIT_0;  // MSB first
         packet <<= 1;
+    } 
+	
+	DMA_SetCurrDataCounter(motor->timerHardware->dmaRef, MOTOR_DMA_BUFFER_SIZE);
+    DMA_Cmd(motor->timerHardware->dmaRef, ENABLE);
+}
+
+void pwmWriteProShot(uint8_t index, uint16_t value)
+{
+    if (!pwmMotorsEnabled) {
```

Comment:
ProShot seems to be reusing the frame format that is defined by DShot. In the interest of maintainability, it's probably better to make both use the same code for this.